core/domain/user_id_migration.py

```diff
@@ -42,6 +44,65 @@
     exp_models.ExplorationSnapshotMetadataModel]
 
 
+def replace_gae_ids(gae_ids):
+    """Replace GAE IDs with user IDs in list.
+
+    Args:
+        gae_ids: list(str). GAE IDs which should be replaced.
+
+    Returns:
+        list(str). New user IDs.
+
+    Raises:
+        MissingUserException: UserSettingsModel with GAE ID doesn't exist.
+    """
+    new_ids = []
+    for gae_id in gae_ids:
+        if gae_id in feconf.SYSTEM_USERS:
+            new_ids.append(gae_id)
+        else:
+            user_settings_model = (
+                user_models.UserSettingsModel.get_by_gae_id(gae_id))
```

Comment:
Since the previous runs of migration didn't use `get_multi` here and there were quite quick anyway, and that the implementation of `get_multi` for the GAE ID will be a bit complicated, I would argue that we do not need it here.